cmd/kubeadm/app/cmd/init.go

```diff
@@ -482,8 +482,8 @@ func (i *Init) Run(out io.Writer) error {
 	}
 
 	// PHASE 7: Make the control plane self-hosted if feature gate is enabled
-	glog.V(1).Infof("[init] feature gate is enabled. Making control plane self-hosted")
 	if features.Enabled(i.cfg.FeatureGates, features.SelfHosting) {
+		glog.V(1).Infof("[init] feature gate is enabled. Making control plane self-hosted")
```

Comment:
This snuck in but I think it's a safe enough change.

The intent was that this only prints if the feature is enabled, but it was always printing.

Reply:
That's ok by me